project/config/hiaredressni/config/views.view.publications_search.yml

```diff
@@ -154,7 +154,7 @@ display:
           exposed: true
           expose:
             operator_id: search_api_fulltext_op
-            label: 'Fulltext search'
```

Comment:
This label should be 'Search publications' as it will add a hidden label for screen readers.

Reply:
Thanks Adam, I have now fixed this.